routes/accounts/register.js

```diff
@@ -1,9 +1,11 @@
 const encryptPassword = require( '../../auth/encryptPassword' )
 const RegistrationEmail = require( '../../src/mail/registration_email' )
+const { createRootItem } = require( '../items/item_response')
 
-const register = ( User, email, password ) =>
+const register = ( User, Item, email, password ) =>
   User.create({ email, password: encryptPassword( password ) })
-    .then( user => RegistrationEmail.send( user ) )
+  	.then( createRootItem(Item) )
+		.then( user => RegistrationEmail.send( user ) )
```

Comment:
Please use spaces instead of tabs on previous two lines, and ensure appropriate indentation. (`.then`s should be left aligned)